lua/nvim-treesitter/indent.lua

```diff
@@ -192,13 +192,34 @@ end
 
 local indent_funcs = {}
 
+local augroup = vim.api.nvim_create_augroup("Treesitter-indents", {})
+local filetypes = {}
+local autocmd_id = nil
+
+local function attach_autocmd(ft)
+  if vim.tbl_contains(filetypes, ft) then
+    return
+  end
+  table.insert(filetypes, ft)
+
+  if autocmd_id then
+    vim.api.nvim_del_autocmd(autocmd_id)
+    autocmd_id = nil
+  end
+
+  autocmd_id = vim.api.nvim_create_autocmd("Filetype", {
+    group = augroup,
+    pattern = filetypes,
+    callback = function()
+      vim.bo.indentexpr = "nvim_treesitter#indent()"
+    end,
+  })
+end
+
 function M.attach(bufnr)
```

Comment:
shouldn't this functionality not handled on module level? The module interface could have a method that runs only once one `setup` or `init_module` or when the module is first attached for a specific filetype

Reply:
i think it would be better, but since it was like this, i did not want to start a refactoring for this, this is just a temporary fix.

Reply:
@theHamsta i removed it since after exploring a bit, i've noticed the autocmd is already set by the configs module.